build/Common.prod.props

```diff
@@ -63,6 +63,8 @@
   <ItemGroup>
     <AdditionalFiles Include=".publicApi\$(TargetFramework)\PublicAPI.Shipped.txt" />
     <AdditionalFiles Include=".publicApi\$(TargetFramework)\PublicAPI.Unshipped.txt" />
+    <None Include=".publicApi\*\PublicAPI.Shipped.txt" />
```

Comment:
@eddynaka The first thing I tried was causing some problems with the PublicAPI file that was used for a build. I added these `None` includes to get the files to show up in VS, left the `AdditionalFiles` alone so the analyzer gets the correct file for build. Seems to work better!